### PR TITLE
Add a version helper tool to amend the intellij plugin release version

### PR DESCRIPTION
**Pull request by testforstephen**

### CODE REVIEW

**Comment by voidfunction** [comment]
Can one of the admins verify this patch?

**Comment by voidfunction** [comment]
@testforstephen Is it a good way to put all those files under folder azure-toolkit-for-intellij directly? I think it may confuse us or new members. How about a new folder (utils or something like that) under PluginsAndFeatures?

**Comment by testforstephen** [comment]
@voidfunction Move it to project tools folder

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/common/CommonConst.java [hunk 1]
```diff
@@ -21,9 +21,12 @@
...
 
+import com.intellij.ide.plugins.PluginManager;
+import com.intellij.openapi.extensions.PluginId;
+
 public class CommonConst {
     public static final String SPARK_SUBMISSION_WINDOW_ID = "HDInsight Spark Submission";
     public static final String PLUGIN_ID = "com.microsoft.tooling.msservices.intellij.azure";
     public static final String PLUGIN_NAME = "azure-toolkit-for-intellij";
-    public static final String PLUGIN_VERISON = "3.0.2";
+    public static final String PLUGIN_VERISON = PluginManager.getPlugin(PluginId.getId(PLUGIN_ID)).getVersion();
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/common/CommonConst.java]: will getPlugin return null?

**Pull request merged**